{
  "config": {
    "step": {
      "user": {
        "data": {
          "username": "Account",
          "password": "Password",
          "mobile_app": "Mobile app",
          "advanced_settings": "Advanced settings"
        },
        "description": "Please enter username and password that you use to connect with the selected Midea mobile application.",
        "title": "Sign-in with Midea app account"
      },
      "advanced_settings": {
        "data": {
          "username": "Account",
          "password": "Password",
          "appkey": "Mobile app key",
          "appid": "Mobile app id",
          "broadcast_address": "Appliance or network broadcast address",
          "use_cloud": "Use cloud API to poll devices",
          "detect_ac_appliances": "Detect air conditioner appliances"
        },
        "description": "You can provide specific Midea mobile app key and app id if the default ones don't work.\n\nYou can also specify network address (e.g. 192.0.1.2) or range (e.g. 192.0.3.4/24) to search for specific appliance(s) if regular discovery doesn't work.",
        "title": "Advanced settings"
      },
      "unreachable_appliance": {
        "data": {
          "ip_address": "IPv4 address",
          "name": "Device name",
          "token": "Token",
          "token_key": "Token key",
          "appliance_action": "Choose action"
        },
        "description": "We were unable to discover an appliance called {name} with the serial number {id}.\n\nPlease choose if you want to exclude appliance, search for it later, use cloud API to poll it, or provide its IPv4 address if you know it.\n\nYou can also provide token and key if you have them. If not, we will try to obtain them from Midea cloud.",
        "title": "Unable to discover appliance"
      },
      "reauth_confirm": {
        "data": {
          "username": "Account",
          "password": "Password",
          "appkey": "Mobile app key",
          "appid": "Mobile app id",
          "broadcast_address": "Appliance or network broadcast address"
        },
        "title": "[%key:common::config_flow::title::reauth%]",
        "description": "The Midea dehumidifier (LAN) integration needs to re-authenticate your account"
      }
    },
    "error": {
      "cannot_connect": "Unable to connect to appliance ({cause}).",
      "connection_error": "Connection error ({cause}).",
      "invalid_auth": "Invalid username or password ({cause})",
      "invalid_ip_address": "Invalid IPv4 address ({cause}), please enter a valid IPv4 address (e.g. 192.0.1.2)",
      "invalid_ip_broadcast": "Invalid IPv4 appliance or broadcast address ({cause}), please enter a valid IPv4 address (e.g. 192.0.1.2 or 255.255.255.255)",
      "no_cloud": "Unable to connect to Midea cloud API",
      "not_discovered": "Unable to find appliance at specified IPv4 address.",
      "midea_client": "An error in communication with Midea API has occurred. See log for more information.",
      "retry_later": "Temporary error with Midea API. Please try later.",
      "unknown": "An unknown error has occurred. See log for more information."
    },
    "abort": {
      "single_instance_allowed": "Already defined a Midea app account. Only a single account is supported for Midea dehumidifier (LAN).",
      "reauth_successful": "Re-authentication was successful",
      "no_configured_devices": "Nothing to configure"
    }
  },
  "options": {
    "step": {
      "init": {
        "description": "Appliance {current_index} of {appliance_count}",
        "data": {
          "exclude": "Ignore checked appliances"
        }
      },
      "appliance": {
        "description": "Appliance {current_index} of {appliance_count}",
        "data": {
          "ip_address": "IPv4 address",
          "name": "Device name",
          "token": "Token",
          "token_key": "Token key",
          "appliance_action": "Choose action"
        }
      }
    }
  }
}